woocommerce/woocommerce
# Fix product SKU styles

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR modifies some `Product SKU` styles.

Before

![Screenshot 2024-11-14 at 3 24 18 PM](https://github.com/user-attachments/assets/8439210f-971d-42c3-aa19-c61aeb7fb333)

After

![Screenshot 2024-11-14 at 3 17 41 PM](https://github.com/user-attachments/assets/5d3f6175-b5c0-40cf-9436-b47d9f614919)

Closes #52396.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to Editor > Single Product template
2. Reset customizations if you've applied any changes
3. Verify Product SKU looks good

![Screenshot 2024-11-14 at 3 20 47 PM](https://github.com/user-attachments/assets/b9df7a28-91a1-44b1-80a7-0fbc094717bb)

After selecting it the `Suffix` should be visible.

![Screenshot 2024-11-14 at 3 17 41 PM](https://github.com/user-attachments/assets/5d3f6175-b5c0-40cf-9436-b47d9f614919)



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
